docs/config.md

```diff
@@ -159,6 +159,8 @@ registry:
 | bearer_token_file | Location of bearer token to be used | N |
 | image_pull_policy | When to pull the image | Y |
 | sandbox_role | Role to give to apb sandbox environment | Y |
+| keep_namespace | Always keep namespace after apb execution | N |
+| Keep_namespace_on_error | Keep namespace after apb execution has an error | N |
```

Comment:
entry should be lowercase `Keep_namespace_on_error` -> `keep_namespace_on_error`